docs/map.json

```diff
@@ -648,6 +653,11 @@
             "id": "library",
             "file": "angular/api-node/generators/library"
           },
+          {
+            "name": "migrate to webpack 5 generator",
```

Comment:
should the name be capitalized? same for line 359 above

Reply:
Didn't see where the capitalized name was, but it was an older version? I kept the casing consistent with the other docs in the list.